external-crates/move/crates/move-compiler/tests/move_2024/parser/macro_identifier_invalid_no_following_characters.exp

```diff
@@ -4,3 +4,12 @@ error[E01002]: unexpected token
 2 │     macro fun foo($
   │                   ^ Expected an identifier following '$', e.g. '$x'
 
+error[E01002]: unexpected token
+  ┌─ tests/move_2024/parser/macro_identifier_invalid_no_following_characters.move:3:1
+  │
+3 │ 
```

Comment:
We need to supress these errors when we find EOF so that we don't get these weird ghost errors